README.md

```diff
@@ -42,7 +39,7 @@ You can run the pipeline locally, or use Snakemake to manage job submission to t
 
 ### How do I get started? ###
 
-1. Create a new working directory that contains the Snakemake file, config file, and files in the envs and scripts directories. If you have git installed the easiest way to get these files is to clone them from Bitbucket. The goal is that you should be able to change just the config.yaml file to analyze data from a new set of samples.
+1. Clone this repository in your working directory. The goal is that you should be able to change just the config.yaml file to analyze data from a new set of samples.
```

Comment:
Clone this repository in your working directory. You will be able to analyze data from a new set of samples just by changing the config.yaml file

Reply:
Done